src/ScriptCs.Core/ScriptInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public class ScriptInfo : IScriptInfo
+    {
+        public ScriptInfo()
+        {
+            LoadedScripts = new List<string>();
+        }
+
+        public string ScriptPath { get; set; }
+        public IList<string> LoadedScripts { get; private set; }
```

Comment:
I don't think we should use Hashset as it does not guarantee ordering. I want the order to be the load order.